tasks/tasks/elwood_processors.py

```diff
@@ -370,6 +374,53 @@ def scale_features(context, filename=None):
         path for path in data_paths_not_str if path not in old_files_normed
     ]
 
+    data_paths_normalized = scaling_core(
+        uuid=uuid,
+        new_files_not_normed=new_files_not_normed,
+        old_files_normed=old_files_normed,
+        scaling_method=scaler.zero_to_one_normalization,
+    )
+
+    # figure out which files paths are have been normalized_robust
+    # and which are new files that are not yet normalized_robust
+    old_files_normed = [
```

Comment:
Break out this list generation into a new function that can be passed the changing file end variable.